src/ui/i18n/translations/common_de.json

```diff
@@ -13,5 +13,6 @@
     "explore": "Erkunden Sie",
     "details": "Siehe Details"
   },
-  "filters": "Filter"
+  "filters": "Filter",
+  "dataverse": "Dataverse"
```

Comment:
Why not just add the text directly in the element since the translations are all the same?

Reply:
Ah yes I see I had misunderstood the conversation, sorry. I guess it could serve a purpose if we want to add translations where 'dataverse' wouldn't work